### PR TITLE
Replaced the lobby menu (with actual art)

### PR DESCRIPTION
**Pull request by ghost**
:cl: Kryson, AMonkeyThatCodes
imageadd: Replaces the lobby menu with sprites instead of HTML
/:cl:

image <-- freeze pass

https://user-images.githubusercontent.com/20987591/130295034-c0ebda62-8465-40ac-932d-865002bfb326.mp4

- [ ] Add minimizing maybe

### CODE REVIEW

**Comment by GoldenAlpharex** [comment]
This looks so sexy, good job!

**Review by Krysonism — APPROVED** [review]
I'm highly biased since this is my own spritework, but i think it looks pretty cool.

**Comment by Mothblocks** [comment]
While the cogwheel takes you to character preferences anyway, can you remove it until I *actually* split game preferences and character preferences into two?

As you know, that split is coming, and I want it to be as obvious as possible how to reach the game settings to existing players.

**Comment by UDaV73rus** [comment]
image

File: code/_onclick/hud/new_player.dm [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+	. = ..()
+	usr.client?.changelog()
+
+/atom/movable/screen/lobby/button/poll
+	icon = 'icons/hud/lobby/bottom_buttons.dmi'
+	icon_state = "changelog"
+	base_icon_state = "changelog"
+	screen_loc = "TOP:-122,CENTER:-26"
+
+	var/new_poll = FALSE
```
Comment (Mothblocks) [inline code/_onclick/hud/new_player.dm:208]: This doesn't appear to be used anymore. Can you add a little notification icon or something?

Comment (Krysonism) [reply code/_onclick/hud/new_player.dm:208]: This has already been sprited.

bild

**Comment by maxymax13** [comment]
whyd u change the background :(

**Comment by cacogen** [comment]
doesn't really fit the style of the rest of the game does it

looks more like advance wars or something

**Review by Mothblocks — CHANGES_REQUESTED** [review]
My two feedbacks

**Comment by Bawhoppen** [comment]
Although very good spritework, I agree it doesn't really fit the style so much. Additionally, it feels beyond odd to have it placed in the top middle. It should probably be in the central middle of the viewport? I know that would obstruct the background though, so that's maybe not workable... For those two reasons, I think it'd be better to just stick with the current menu system that is visually neutral and un-intrusive.

File: code/_onclick/hud/new_player.dm (hunk 1 above)
Comment (AMonkeyThatCodes) [reply code/_onclick/hud/new_player.dm:208]: yeah didnt implement this yet

**Comment by AMonkeyThatCodes** [comment]
@Mothblocks I commented out the settings menu, I can remove it completely but it seems like a waste of everyones time

**Comment by UzedPickle** [comment]
it's unclear if the ready buttons readys me up or indicates if I'm ready or not, this doesn't matter to me since i know how it works but it could be confusing for a new player

**Comment by AMonkeyThatCodes** [comment]
@Krysonism i think the above comment has a good point, do you think we could improve that?

**Comment by Krysonism** [comment]
> 
> 
> @Krysonism i think the above comment has a good point, do you think we could improve that?

I think even a first time player as a good chance of figuring out. 

If you have good idea on how implementing it without deteriorating the aesthetics im willing to give it a shot but i don't think it is a big issue.

**Comment by AMonkeyThatCodes** [comment]
not really, lets just try it out like this for now.

**Pull request merged**